src/cyclonedds/setup.py

```diff
@@ -52,6 +52,11 @@
     ],
     packages=find_packages(exclude=("tests", "examples")),
 	ext_modules = [ddspy],
-    scripts=['tools/ddsls.py'],
+    entry_points = {
+    	"console_scripts": [
+    				"ddsls=cyclonedds.tools.ddsls:command",
+    				"pubsub=cyclonedds.tools.pubsub:command"
+    			     ],
+    },
```

Comment:
@thijsmie, mentioned this will generate the proper shims, so definitely very welcome :+1:. Unfortunately, this'll probably conflict with some of the changes I did for scikit-build.